sql/src/main/java/io/crate/analyze/relations/DocTableRelation.java

```diff
@@ -137,7 +139,7 @@ private void ensureColumnCanBeUpdated(ColumnIdent ci) {
         }
     }
 
-    private void ensureNotUpdated(ColumnIdent columnUpdated, ColumnIdent protectedColumnIdent, String errorMessage) {
+    private void ensureNotUpdated(ColumnIdent columnUpdated, ColumnIdent protectedColumnIdent, String errorMessage) throws ColumnValidationException {
```

Comment:
only non-runtime exceptions must be declared

Reply:
sure, stupid me ...